db/seeds.rb

```diff
@@ -1,37 +1,33 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
 require "factory_girl"
+require 'csv'
 
-RandomWord.exclude_list << /.*_.*/
-
-def generate_item_name
-  "#{RandomWord.adjs.next} #{RandomWord.nouns.next} of #{RandomWord.nouns.next}".titleize
-end
-
-rarity_names = ["junk", "heirloom", "antiquity", "relic"]
-values = [100, 250, 500, 1000]
+descriptor1 = CSV.read("app/assets/csv/descriptor1.csv")
+ordinary = CSV.read("app/assets/csv/ordinary.csv")
+heirloom = CSV.read("app/assets/csv/heirloom.csv")
+antiquity = CSV.read("app/assets/csv/antiquity.csv")
+descriptor2 = CSV.read("app/assets/csv/descriptor2.csv")
 
 250.times do
-  Item.create!(name: generate_item_name, rarity: rarity_names[0], value: values[0])
+  Item.create(name: "#{descriptor1[0].sample.titleize}
+    #{ordinary[0].sample.titleize} of #{descriptor2[0].sample.titleize}",
+    rarity: "ordinary", value: 100 )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.